src/components/sections/ProductShelf/product-shelf.scss

```diff
@@ -1,35 +1,50 @@
 @import "src/styles/scaffold";
 
-[data-product-shelf-items] {
+/* stylelint-disable */
+
+[data-fs-product-shelf] {
+  // --------------------------------------------------------
+  // Design Tokens for Product Shelf
+  // --------------------------------------------------------
+
+  --fs-product-shelf-height:                      21.06rem;
+  --fs-product-shelf-margin-top:                  calc(-1 * var(--fs-spacing-0));
+  --fs-product-shelf-margin-bottom:               calc(-1 * var(--fs-spacing-3));
+
+  --fs-product-shelf-items-gap:                   var(--fs-spacing-2);
```

Comment:
Looked if we had a `--fs-grid-gap-*` for this, but the first one,`0`, is already using `--fs-spacing-3`. 

Maybe we can standardize the gaps better later, I know the product grid on mobile PLP has another one that is not mapped by a token (2px).

Reply:
That's true! 
I just created new `--fs-grid-gap-x` values and adapt the older values. 
About the 2px from the `product-grid` I don't know the best approach to deal with it since we don't have 2px value as global token since `--fs-spacing-0: 0.25rem (4px)`. @renatamottam , any ideas?

Reply:
Nice improvement! I just think we should only list the spaces we really use.
For example: in this list below, we do not use  `--fs-grid-gap-0` and `--fs-grid-gap-1`
```
--fs-grid-gap-0                      : var(--fs-spacing-0);
--fs-grid-gap-1                      : var(--fs-spacing-1);
--fs-grid-gap-2                      : var(--fs-spacing-2);
--fs-grid-gap-3                      : var(--fs-spacing-3);
--fs-grid-gap-4                      : var(--fs-spacing-4);
--fs-grid-gap-5                      : var(--fs-spacing-5);
```

Reply:
Hey @filipewl , I paired along with @renatamottam and we tweak the grid-gaps for the `product-grid.scss` as well. Now we only list the gaps that we really use as global grid tokens.